src/mono/mono/mini/aot-compiler.c

```diff
@@ -8513,8 +8534,10 @@ mono_aot_parse_options (const char *aot_options, MonoAotOptions *opts)
 			mini_debug_options.no_seq_points_compact_data = FALSE;
 			opts->gen_msym_dir = TRUE;
 			opts->gen_msym_dir_path = g_strdup (arg + strlen ("msym_dir="));
-		} else if (str_begins_with (arg, "direct-pinvoke")) {
-			opts->direct_pinvoke = TRUE;
+		} else if (str_begins_with (arg, "direct-pinvokes")) {
+			opts->direct_pinvokes = g_list_append (opts->direct_pinvokes, g_strdup (arg + strlen ("direct-pinvokes=")));
```

Comment:
This should be begins_with (arg, "direct-pinvokes=")

Reply:
Changed!